src/signals/incident-management/containers/Dashboard/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { useCallback, useEffect, useRef, useState } from 'react'
+
+import { useClickOutside } from '@amsterdam/asc-ui'
+import { useForm, FormProvider } from 'react-hook-form'
+import { generateParams } from 'shared/services/api/api'
+
+import SelectList from './SelectList'
+import { FilterContainer } from './styled'
+import type { Option } from './types'
+
+type Props = {
+  callback?: (queryString: string) => void
+}
+
+export const Filter = ({ callback = () => {} }: Props) => {
+  const [filterActiveName, setFilterActiveName] = useState('')
```

Comment:
Its name because it are names of the form. A slug I associate with record (like from a database). Name derives from useFilters giving a config to render a form that has form fields that have name properties.